core/lib/default_da_clients/src/object_store/client.rs

```diff
@@ -0,0 +1,84 @@
+use std::{fmt::Debug, sync::Arc};
```

Comment:
Nit: `fmt::Debug` import is unnecessary; it's only used as a derive macro, which is a part of the `std` prelude anyway.